torch_glow/src/PyTorchModelLoader.cpp

```diff
@@ -1045,6 +1059,34 @@ llvm::Error PyTorchModelLoader::loadPRelu(const torch::jit::Node *ptNode) {
   return addGlowNodeValue(outputs[0], glowNode);
 }
 
+llvm::Error PyTorchModelLoader::loadFlatten(const torch::jit::Node *ptNode) {
+  auto inputs = ptNode->inputs();
+  auto outputs = ptNode->outputs();
+  RETURN_IF_ERR(checkInputAndOutputSizes(inputs, 3, outputs, 1));
+
+  glow::NodeValue in;
+  ASSIGN_VALUE_OR_RETURN_ERR(in,
+                             getGlowNodeValue(inputs[FlattenInputs::input]));
+
+  auto startHandle = glow::Handle<int32_t>::createInvalidHandle();
+  ASSIGN_VALUE_OR_RETURN_ERR(
+      startHandle,
+      getGlowConstantHandle<int32_t>(inputs[FlattenInputs::start_dim]));
+  uint32_t startDim;
```

Comment:
from the docs, this should default to 0 and endDim should default -1 https://pytorch.org/docs/stable/torch.html#torch.flatten